index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Book List</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <div class="navContaner">
+          <h3>Awesome Books</h3>
+          <ul class="spans">
+            <li class="liborder"><span class=" spanLiL">List</span></li>
+            <li class="liborder"><span class="spanLiA">Add new</span></li>
+            <li><span class="spanLiC">contact</span></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="date"></div>
+    </header>
+    <section class="hero">
+      <div class="asesomeBooks d-none">
+        <h1>All awesome Books</h1>
+        <div class="booksContaner">
+          <div class="books">
+            <ul class="book-list"></ul>
+          </div>
+        </div>
+      </div>
+      <div class="addForm d-none">
+        <h1>Add a new Book</h1>
```

Comment:
- [ ] Please, consider using the `h1` tag one time in your code, all other headers should be `hx`, aka h2, h3, ... `h1` is talking for browser what about is your page.